packages/datatrak-web/src/api/CurrentUserContext.tsx

```diff
@@ -27,10 +26,6 @@ export const CurrentUserContextProvider = ({ children }: { children: React.React
     return <FullPageLoader />;
   }
 
-  if (currentUserQuery.isError) {
-    return <ErrorDisplay title="Error loading user" error={currentUserQuery.error as Error} />;
```

Comment:
Just checking, are there ever going to be cases where this is still needed? E.g. some other kind of error?

Reply:
Tbh I'm not sure, for instance the request could timeout or something. But this is true of any request and I feel the standard behaviour of just showing a toast error is cleaner than what we do here.

Reply:
I think what I mean here is that what this `ErrorDisplay` does is hides all content in case there are some things that are visible. I think it also solved an issue with re-rendering/infinite loops. Potentially could we instead redirect to the login page for all user errors?